zeebe-cluster/values.yaml

```diff
@@ -38,14 +29,70 @@ service:
     port: 26501  
   internal:
     port: 26502
-resources: 
-  requests:
-    cpu: 500m
-    memory: 1Gi
-  limits:
-    cpu: 1000m
-    memory: 2Gi
-global: 
-  elasticsearch:
-    host: "elasticsearch-master"
-    port: 9200 
+
+resources:  {}
+#   requests:
+#     cpu: 500m
+#     memory: 1Gi
+#   limits:
+#     cpu: 1000m
+#     memory: 2Gi
+
+data:
+  enabled: false
```

Comment:
@ysaakpr why enabled false? what is this supposed to do?

Reply:
No, will make it default true